api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/processor/DetermineEntitlementMessageProcessorTest.java

```diff
@@ -123,7 +125,7 @@ void shouldSuccessfullyProcessMessageAndTriggerPaymentWhenClaimantIsEligible() {
 
     @ParameterizedTest(name = "Qualifying benefit status={0}")
     @ValueSource(strings = {"CONFIRMED", "NOT_CONFIRMED"})
-    void shouldExpireClaimAndSendNoLongerOnSchemeEmailWhenIneligibleWithNoChildrenAndNotPregnant(
+    void shouldExpireClaimAndSendNoLongerOnSchemeEmailAndSetCardStatusToPendingCancellationWhenIneligibleWithNoChildrenAndNotPregnant(
```

Comment:
This name is getting to be too long now. I think we should go with `shouldExpireClaimWhenIneligibleWithNoChildrenAndNotPregnant` and either update the name in the `@ParameterizedTest` annotation or add a comment. We can't keep adding every change to the method name.

Reply:
Renamed to be shorter. Didn't add a comment seeing as the tests themselves are descriptive of the expected behaviour.